content/infrastructure/installation/install-rippled-on-ubuntu.md

```diff
@@ -91,6 +89,28 @@ Before you install `rippled`, you must meet the [System Requirements](system-req
         sudo setcap 'cap_net_bind_service=+ep' /opt/ripple/bin/rippled
 
 
+9. Optional: configure core dumps
+
+    By default Ubuntu is not configured to produce core files useful for debugging crashes.
+    First run:
+    
+            ulimit -c unlimited
+    
+    Now run `sudo systemctl edit rippled`. The default editor should open and add
+
+        [Service]
+        LimitCORE=infinity
+
+    This creates `/etc/systemd/system/rippled.service.d/override.conf` saving core dumps while not interfering 
+    with the service file provided by the `rippled` package.
+    Core dumps appearing in `/var/lib/apport/coredump/` can be loaded into `gdb` with
+
+        gdb <path_to_rippled> <path_to_core_file>
+
+    provided the `rippled-dbgsym` package has been installed. 
+    You will need permission to read files in the core dump directory.
```

Comment:
In addition to not hardwrapping lines, let's also not continue sentences from before the code block. Instead:

```suggestion
    **Note:** To debug a core file this way, you must have the `rippled-dbgsym` package installed, and you need permission to read files in the core dump directory.
```